crates/bevy_render/src/camera/mod.rs

```diff
@@ -23,14 +20,13 @@ impl Plugin for CameraPlugin {
         app.register_type::<Camera>()
             .register_type::<Viewport>()
             .register_type::<Option<Viewport>>()
-            .register_type::<Visibility>()
-            .register_type::<ComputedVisibility>()
-            .register_type::<VisibleEntities>()
+            .register_type::<Range<f32>>()
+            .register_type_data::<Range<f32>, ReflectSerialize>()
+            .register_type_data::<Range<f32>, ReflectDeserialize>()
```

Comment:
These are already registered by the `CorePlugin`

Reply:
I figured it would be best to leave them in here in case someone was using `bevy_render` apart from `bevy_core`, but I just realized `bevy_core` is one of `bevy_render`'s dependencies 😅

I'll remove these registrations.